controllers/networkfence_controller.go

```diff
@@ -137,7 +137,7 @@ func (r *NetworkFenceReconciler) Reconcile(ctx context.Context, req ctrl.Request
 	}
 	nwFence.Status.Result = csiaddonsv1alpha1.FencingOperationResultSucceeded
 	nwFence.Status.Message = "NetworkFence operation succeeded"
-	if err := r.Client.Update(ctx, nwFence); err != nil {
+	if err := r.Client.Status().Update(ctx, nwFence); err != nil {
```

Comment:
correct same here also https://github.com/csi-addons/kubernetes-csi-addons/blob/c037bfe5f369c5a0e2bc792bb872615756c75d83/controllers/csiaddonsnode_controller.go#L106-L121? maybe as a follow-up PR?

cc @Rakshith-R

Reply:
Sure, I'll send a PR for that too :+1:

Reply:
Sent :https://github.com/csi-addons/kubernetes-csi-addons/pull/76

Reply:
> Sure, I'll send a PR for that too +1

Thanks !